ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.js

```diff
@@ -0,0 +1,265 @@
+import React, { useContext, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import Button from '../../../components/ui/button';
```

Comment:
Should be able to use the `Button` component from `component-library` instead of this one.